storage.bs

```diff
@@ -331,8 +351,26 @@ must run these steps:
 
    <li><p>Let <var>quota</var> be <a>site storage quota</a> for <var>origin</var>.
 
-   <li><p>Let <var>dictionary</var> be a new {{StorageEstimate}} dictionary whose {{usage}} member
-   is <var>usage</var> and {{quota}} member is <var>quota</var>.
+   <li><p>Let <var>applicationCache</var> be <a>application cache site storage usage</a>
+   for <var>origin</var>.
+
+   <li><p>Let <var>indexedDB</var> be <a>indexedDB site storage usage</a> for <var>origin</var>.
+
+   <li><p>Let <var>caches</var> be <a>caches site storage usage</a> for <var>origin</var>.
+
+   <li><p>Let <var>serviceWorkerRegistrations</var> be <a>service worker registration
+   site storage usage</a> for <var>origin</var>.
+
+   <li><p>Let <var>usageDetails</var> be a new {{StorageUsageDetails}} dictionary
```

Comment:
This step is weird because you say to create the dictionary with all these things set, but then you say "actually, exclude them sometimes".

It would be better to phrase this informatively. Roughly,

1. Let _usageDetails_ be a new `StorageUsageDetails` dictionary.
1. If _applicationCache_ > 0, then set _usageDetails_["`applicationCache`"] to _applicationCache_.
1. If...